game/templates/game/landing.html

```
@@ -102,6 +102,8 @@
         </div>
       </div>
       <div class="nav-right">
+        <button id="theme-toggle" class="theme-toggle">🌙</button>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)